### DISCUSSION TITLE
Is `UseLegacy` still active?

### DISCUSSION THREAD

**Discussion by guettli** [topolvm/topolvm#899] [Q&A] [answered]
A lot of places in the cod use `UseLegacy`. 

Is that special case still needed?

**Comment by satoru-takeuchi** [comment, marked as answer]
> Is that special case still needed?

Yes, it's necessary.

**Reply by guettli** [reply]
Thank you for your reply.

Do you have a rough estimation how long you want to support the legacy mode?

**Reply by satoru-takeuchi** [reply]
I don't plan removing this flag for now. My company uses topolvm. We should suspend our service on the migration from legacy mode to the new one and we can't accept this option at present. I guess the same thing can be said for many long-term topolvm users.

One day we might decide to remove this mode for some reasons. Then we will announce this decision and will wait for some versions before doing so.
